app/stores/CollectionsStore.ts

```diff
@@ -162,6 +165,31 @@ export default class CollectionsStore extends BaseStore<Collection> {
     }
   }
 
+  @action
+  fetchPublicCollections = async (
```

Comment:
Good news – this extra method isn't needed. All collections are guaranteed to be available on the client. But you could create a getter for public collections, something like:

```javascript
@computed
get publicCollections() {
  return this.orderedData.filter(collection => ["read", "read_write"].includes(collection.permission));
}
```

Reply:
It was intended for the case when we open the details page directly. Let's make the usual `collections.list` request and then use the `publicCollection` to get the collections. What do you think?

Reply:
Agreed